src/docs/patterns/layout.mdx

```diff
@@ -26,10 +26,10 @@ You can see examples of how to do various layouts on the Angular Flex Layout web
 
 ## React
 
-Material-UI for React applications provides a built-in [layout](https://material-ui.com/layout/basics/) mechanism, based on CSS flexbox. It allows you to build layouts based on CSS Grids, standard flexbox CSS, or a combination. Some of the main benefits of this library are:
+Material-UI for React applications provides a built-in [layout](https://mui.com/layout/basics/) mechanism, based on CSS flexbox. It allows you to build layouts based on CSS Grids, standard flexbox CSS, or a combination. Some of the main benefits of this library are:
 
 -   Built-in Grid components, based on Material Design 12-column layout
 -   Custom flexbox rules for more control
 -   Responsive layout capability (e.g., show/hide elements, dynamic styling rules, etc. based on screen size)
 
-You can see examples of how to do various Grid layouts on the [Material UI](https://material-ui.com/layout/grid/) website.
+You can see examples of how to do various Grid layouts on the [Material UI](https://mui.com/layout/grid/) website.
```

Comment:
This one auto redirects to https://mui.com/components/grid/ which looks good with me